docs/components/base/_index.md

```diff
@@ -405,6 +405,71 @@ myBase.Stop(context.Background())
 {{% /tab %}}
 {{< /tabs >}}
 
+### GetProperties
+
+Get the width and turning radius of the base in meters.
+
+{{< alert title="Tip" color="tip" >}}
+These properties are inherent to the {{< glossary_tooltip term_id="model" text="model" >}} of base, so you do not need to [configure](#configuration) them.
+{{< /alert >}}
+
+{{< tabs >}}
+{{% tab name="Python" %}}
```

Comment:
GetProperties is only supported in Go currently.

Reply:
https://python.viam.dev/autoapi/viam/components/base/client/index.html#viam.components.base.client.BaseClient.get_properties

Reply:
Oh that's surprising. https://viam.atlassian.net/browse/RSDK-3321 is still in triage, but it does look like this was implemented. @cheukt can you clarify if GetProperties on a base is supported in python?

Reply:
base.get_properties is supported currently, it was done as part of a different ticket.

Reply:
Thank you!